concepticondata/conceptrelations.tsv

```diff
@@ -1217,3 +1217,19 @@ SOURCE	SOURCE_GLOSS	RELATION	TARGET	TARGET_GLOSS
 3074	FIX	instanceof	2243	REPAIR
 1755	UNDRESS	instanceof	3538	TAKE OFF
 158	PUT ON	antonymof	3538	TAKE OFF
+3603	EARTH OR LAND	instanceof	1228	LAND
+3603	EARTH OR LAND	instanceof	626	LAND
+3604	LIGHT BLUE	instanceof	837	BLUE
+3604	LIGHT BLUE	narrower	837	BLUE
+3605	LIGHT GREEN	instanceof	1425	GREEN
+3605	LIGHT GREEN	narrower	1425	GREEN
+3606	STURGEON	instanceof	227	FISH
+3606	STURGEON	narrower	227	FISH
+3607	MOCCASIN	instanceof	1381	SHOE
+3607	MOCCASIN	narrower	1381	SHOE
+3609	TOWN OR VILLAGE	instanceof	1391	TOWN
```

Comment:
this is obviously a narrower relation, no instanceof relation. Note that this passes the tests, because the tests do not test instanceof.

Reply:
Doesn't narrower include both instanceof and partof? And it is an instanceof (i.e. hyponymic) relation by the definition we have (i.e. "Subcategory of x", the opposite of "hyperonymof").